File: sdkwork-ai-sdk-csharp/Models/Usage.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class Usage
    {
        public int? PromptTokens { get; set; }
        public int? CompletionTokens { get; set; }
        public int? TotalTokens { get; set; }
        public PromptTokensDetails? PromptTokensDetails { get; set; }
        public CompletionTokensDetails? CompletionTokensDetails { get; set; }
    }
}

File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/Usage.kt
package com.sdkwork.ai

data class Usage(
    val promptTokens: Int? = null,
    val completionTokens: Int? = null,
    val totalTokens: Int? = null,
    val promptTokensDetails: PromptTokensDetails? = null,
    val completionTokensDetails: CompletionTokensDetails? = null
)
